src/Stateless/ParameterConversion.cs

```diff
@@ -6,8 +6,6 @@ static class ParameterConversion
     {
         public static object Unpack(object[] args, Type argType, int index)
         {
-            Enforce.ArgumentNotNull(args, nameof(args));
```

Comment:
Looking up the call-chain, it looks like args can never be null, but it can be a zero length array. So in the current state it shouldn't require a null check. But I'll add one for future proofing.